var/spack/repos/builtin/packages/nicstat/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import os
+
+
+class Nicstat(Package):
```

Comment:
This should be `MakefilePackage` base class.

Reply:
fix this in https://github.com/spack/spack/pull/14173/commits/14b6d91a65b36d3a1155901985eae15554ecec3a